ui/admin/app/components/form/policy/policy-selection/index.hbs

```diff
@@ -0,0 +1,75 @@
+<Hds::Form::Fieldset @isRequired={{true}} class='policy' as |F|>
+
+  <F.Legend>{{t (concat 'resources.policy.form.' @name '.label')}}</F.Legend>
+  <F.HelperText>
+    {{t (concat 'resources.policy.form.' @name '.help')}}
+  </F.HelperText>
+
+  <F.Control>
+    <Hds::Form::Select::Field
+      name={{@name}}
+      disabled={{@disabled}}
+      aria-label={{t (concat 'resources.policy.form.' @name '.label')}}
+      @width='80%'
+      {{on 'change' @onSelect}}
+      as |F|
+    >
+      <F.Options>
+        <option disabled hidden selected value=''>
+          {{t 'titles.choose-an-option'}}
+        </option>
+        {{#each-in @options as |key value|}}
+          <option
+            value={{value}}
+            selected={{eq key @selectedOption}}
+            disabled={{@forceDoNotDelete}}
+          >
+            {{t
+              (concat 'resources.policy.form.' @name '_options.' key '.label')
+            }}
+          </option>
+        {{/each-in}}
+      </F.Options>
+
+    </Hds::Form::Select::Field>
+
+  </F.Control>
+</Hds::Form::Fieldset>
+
+{{#if @isCustomInputVisible}}
+  <Hds::Form::Field class='policy-custom-input' @layout='vertical' as |F|>
+    <F.Label>{{t (concat 'resources.policy.titles.' @customInputName)}}
+    </F.Label>
+    <F.Control>
+      <Hds::SegmentedGroup as |S|>
+        <S.TextInput
+          @value={{@inputValue}}
+          disabled={{@disabled}}
+          aria-label={{t 'resources.policy.label'}}
```

Comment:
What translation is this referring to, I don't think this is pointing to the correct label.